earth_enterprise/rpms/buildSrc/src/main/groovy/GeeDeb.groovy

```diff
@@ -82,4 +96,24 @@ class GeeDeb extends com.netflix.gradle.plugins.deb.Deb {
 
         findRequires(packageInputFiles).each { requires(it) }
     }
-}
\ No newline at end of file
+
+    // Formats a multi-line description to be compatible with a Debian build
+    void formatPackageDescription(String descrip){
```

Comment:
@mrstephen: I'm commenting on the latest pull request commit, because the earlier comments became outdated, are would be harder to find.

Yes, in Groovy, basically, all expressions to access an object's member variable are implicit calls to getters and setters.  You can have a look at http://groovy-lang.org/style-guide.html#_getters_and_setters.  It uses the same accessor convention used in Java for accessing Java bean properties.  The getter for `propertyName` is the method called `getPropertyName` and the setter is called `setPropertyName`.  (If the property is a boolean, you can use `isProperty` instead of `getProperty`.)

I was thinking that it would be more in line with configuring Gradle tasks to use a property, like having another version of `packageDescription`.  However, this is a stylistic choice, so I'll approve even if it's implemented with a method that's not an setter.